README.md

```diff
@@ -37,35 +43,32 @@ VeraConfiguration.Builder veraBuilder = new VeraConfiguration.Builder
         .setHideHeader(false) // A boolean value for the SDK to hide or show the default header. Defaults to `false`
         .setDeeplinkPrefix(veraDomain) //A string value defining your custom "deepLinkPrefix". Used when generating deep links that you can pass into Vera and should open your app.
         .setLanguage(sdkLanguage) // Choose Vera language. Defaults to `en` (English)
-        
         .onRequestRefreshToken(this::fetchUserToken) // Listener triggered when user token need refresh
         .onMessageListener(this::onMessage) // Listener to messages from Vera and implement logic for them
         .onCloseListener(this::closeVeraSdk) // Listener triggered when Vera closed
 ```
 
-You can start Vera with user data :
-
-```java
-
-     builder.startWithLogin(useName(), userId, token);
-```
-
-You can also start Vera without user data, i.e. anonymously:
-
-```java
-
-     builder.startWithoutLogin();
-```
+2.  Start sdk
+   -  Start Vera with user data:</br>
+     `veraBuilder.startWithLogin(userName(), userId, token);`
+   - Start Vera without user data, i.e. anonymously:</br>
+    `veraBuilder.startWithoutLogin();`
+    
+## Bi-directional Communication
+The SDK implements bi-directional communication between the Vera platform and the client application.
```

Comment:
You may want to explain what you mean here, e.g. how the communication works in each direction